react-native/android/analytics.gradle

```diff
@@ -57,8 +57,9 @@ class SendAnalyticsTask extends DefaultTask {
     def sendAnalytics() {
        try {
             def env = System.getenv()
-            def disableAnalytics= env['REALM_DISABLE_ANALYTICS']
-            if (disableAnalytics == null || disableAnalytics != "true") {
+            def disableAnalytics = env['REALM_DISABLE_ANALYTICS'] != null
```

Comment:
```suggestion
            def disableAnalytics = env['REALM_DISABLE_ANALYTICS'] != null || env['REALM_DISABLE_ANALYTICS'] == "true"
```

Reply:
`!= null` is already stronger than `== true`. If `REALM_DISABLE_ANALYTICS` is set to `true`, then it won't be null.